Snippets/Snippets_6/Callback/Usage.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Snippets6.Callback
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    public class Usage
+    {
+        public async Task Describe()
+        {
+            IBus bus = null;
+            var options = new SendOptions();
+
+            #region CallbackWithMessageAsResponse
+            var responseMessage = await bus.RequestWithTransientlyHandledResponseAsync<PlaceOrderResponse>(new PlaceOrder(), options);
```

Comment:
@danielmarbach i think that advice is out of date since 

> it of course also helps with the case where functionality is exposed with both synchronous and asynchronous methods such that you need a name difference to distinguish them

Is now generally recognized as a antipattern